taskcluster/ac_taskgraph/loader/build_config.py

```diff
@@ -125,7 +125,7 @@ def loader(kind, path, config, params, loaded_tasks):
     # Filter away overridden jobs that we wouldn't build anyways to avoid ending up with
     # partial job entries.
     overridden_jobs = {k: v for k, v in config.pop('overriden-jobs', {}).items() if affected_components is ALL_COMPONENTS or k in affected_components}
-    jobs = merge(jobs, config.pop('overriden-jobs', {}))
```

Comment:
The line just above pops `overriden-jobs` (there's a typo on this key, by the way), which means this line always resolves to `merge(jobs, {})` and that's why the bug came up.